src/LUNA_architecture_tests/LUNA_architecture_check_series1.py

```diff
@@ -0,0 +1,154 @@
+import sqlite3
+import sys; sys.path.insert(0, '../..')
+from autograd import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import time
+from IPython.display import display
+
+from src.models import LUNA
+from src.utils import generate_data
+import LUNA_database_parse as par
+
+def run_experiments():
+    x, y, _ = generate_data(number_of_points=50, noise_variance=9)
```

Comment:
Due to the change in function definition, it has to be: `x, y, _, _ = generate_data...`

Reply:
oh u right lemme fix

Reply:
thats odd I was able to run it

Reply:
It's perhaps because your version of `utils.py` is not up to date.